example/listenvironments/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/google/go-github/v42/github"
+	"golang.org/x/oauth2"
+)
+
+// An example of how to use ListEnvironments method with EnvironmentListOptions.
+// It's runnable with the following command:
+// export GITHUB_TOKEN=your_token export GITHUB_REPOSITORY_OWNER=your_owner export GITHUB_REPOSITORY_NAME=your_repo && go run .
+func main() {
+
+	var client *github.Client
+	var ctx = context.Background()
+
+	token := os.Getenv("GITHUB_AUTH_TOKEN")
+	repo := os.Getenv("GITHUB_REPOSITORY_NAME")
+	owner := os.Getenv("GITHUB_REPOSITORY_OWNER")
+
+	if token == "" {
+		log.Fatal("Unauthorized: No token present")
+	}
+
+	ts := oauth2.StaticTokenSource(&oauth2.Token{AccessToken: token})
+	tc := oauth2.NewClient(ctx, ts)
+	client = github.NewClient(tc)
```

Comment:
```suggestion
	client := github.NewClient(tc)
```

And then lines 17-18 above can be deleted.

Reply:
Updated and removed the var declaration